### PR TITLE
ci(docs-infra): run tests against local Angular packages too

### PR DESCRIPTION
**Pull request by gkalpak**
This allows catching regressions earlier.

### CODE REVIEW

File: aio/src/app/custom-elements/contributor/contributor.service.spec.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ describe('ContributorService', () => {
     it('contributors observable should complete', () => {
       let completed = false;
       contribService.contributors.subscribe(undefined, undefined, () => completed = true);
-      expect(true).toBe(true, 'observable completed');
+      expect(completed).toBe(true, 'observable completed');
```
Comment (petebacondarwin) [inline aio/src/app/custom-elements/contributor/contributor.service.spec.ts]: ❤️

File: scripts/ci/install.sh [hunk 2]
```diff
@@ -49,15 +49,13 @@ travisFoldEnd "bower-install"
...
+  ${CI_MODE} == "aio_local" ||
   ${CI_MODE} == "aio_e2e" ||
   ${CI_MODE} == "aio_tools_test"
 ]]; then
   # angular.io: Install all yarn dependencies according to angular.io/yarn.lock
   travisFoldStart "yarn-install.aio"
     (
-      # HACK (don't submit with this): Build Angular
-      ./build.sh --packages=compiler,core,elements --examples=false
-
```
Comment (petebacondarwin) [inline scripts/ci/install.sh]: LOL

File: aio/src/app/shared/copier.service.ts [hunk 3]
```diff
@@ -34,7 +35,7 @@ export class CopierService {
       this.fakeElem.style[ isRTL ? 'right' : 'left' ] = '-9999px';
 
       // Move element to the same position vertically
-      const yPosition = window.pageYOffset || document.documentElement.scrollTop;
+      const yPosition = window.pageYOffset || docElem.scrollTop;
```
Comment (petebacondarwin) [inline aio/src/app/shared/copier.service.ts]: I get the feeling that this change should be in a different commit ... 🤔

**Review by petebacondarwin — APPROVED** [review]
LGTM - but I wonder if we should be adding new jobs to CircleCI rather than Travis?
Or are we just not in a position to build AIO via CircleCI yet?

File: scripts/ci/install.sh (hunk 2 above)
Comment (IgorMinar) [reply scripts/ci/install.sh]: omg! :)

**Review by IgorMinar — APPROVED** [review]

File: aio/src/app/shared/copier.service.ts (hunk 3 above)
Comment (gkalpak) [reply aio/src/app/shared/copier.service.ts]: 👍 Agreed :grin:
For the record, this was failing because of the updated `lib.dom.d.ts` typings in latest TS (which is brought in with `aio-use-local`).

**Comment by gkalpak** [comment]
WRT Travis vs CircleCI:
This new job is very similar to the `aio` job. I think it makes sense to keep them on the same CI.
(Off the top of my head, there shouldn't be anything blocking us from moving the angular.io-related jobs to CircleCI.)

File: aio/src/app/shared/copier.service.ts (hunk 3 above)
Comment (gkalpak) [reply aio/src/app/shared/copier.service.ts]: I moved it to a new commit, then it felt appropriate to upgrade TS to 3.1.x (which this change is related to), which in turn required upgrading Angular to 7.0.0-rc.0 :grin:

Now you have to review again 😛

**Comment by gkalpak** [comment]
Rebased on master and added a new commit (per https://github.com/angular/angular/pull/26202/commits/b322d4c26baa821bf7e1dc1e5cbafcb8e7df446f#r222719118 - also took the chance to upgrade Angular and TS).

@petebacondarwin, @IgorMinar: Please, re-approve 🙏 (FWIW, only the last commit is new.)

**Review by petebacondarwin — APPROVED** [review]

**Comment by jasonaden** [comment]
@gkalpak Can you create a separate PR for patch? Specifically because of the changes to `package.json` and `yarn.lock` as these will never merge cleanly.

**Comment by gkalpak** [comment]
Rebased this on master and changed PR target to `master-only`.
Created #26306 for patch.
